src/SAWScript/Crucible/JVM/Override.hs

```diff
@@ -243,13 +243,15 @@ methodSpecHandler opts sc cc top_loc css h = do
                 case res of
                   Left (OF loc rsn)  ->
                     -- TODO, better pretty printing for reasons
-                    liftIO $ Crucible.abortExecBecause
-                      (Crucible.AssumedFalse (Crucible.AssumptionReason loc (show rsn)))
+                    liftIO
+                      $ Crucible.abortExecBecause
+                      $ Crucible.AssertionFailure
+                      $ Crucible.SimError loc
+                      $ Crucible.AssertFailureSimError "assumed false" (show rsn)
```

Comment:
This isn't an assertion error, so we should just use the `InfeasibleBranch` reason.

Reply:
Humm, sorry, looking into it a bit more, I guess this does make sense to consider as an assertion failure.